internal/command/action_v2_execution.go

```diff
@@ -201,7 +201,7 @@ func (t SetExecution) GetTargets() []string {
 }
 
 func (e *SetExecution) IsValid() error {
-	if len(e.Targets) == 0 {
+	if e.Targets == nil {
```

Comment:
add a comment why we check a slice for nil

Reply:
Actually, this is never true.
Passing an empty slice is fine now, as it basically deletes the execution.
I removed the validation altogether.